source/image-handler/index.js

```diff
@@ -22,9 +22,14 @@ exports.handler = async (event) => {
         const request = await imageRequest.setup(event);
         console.log(request);
         const processedRequest = await imageHandler.process(request);
+        const headers = getResponseHeaders();
+        headers["Content-Type"] = request.ContentType;
```

Comment:
What if you are changing the image from a jpg to a png. 

If i put 
`'toFormat' => 'png'`
in my image settings but the source is a jpeg this will output image/jpg